verifier/data/tests/README.md

```diff
@@ -8,12 +8,14 @@
   currently used by Intel.
 * `leaf_cert.pem` - Leaf of a certificate chain. This is a copy of an Intel
   leaf certificate which was in an actual hardware quote.
-* `root_crl.pem` - CRL for the root CA of a certificate chain. This was
+* `root_crl.der` - CRL for the root CA of a certificate chain. This was
```

Comment:
The x509-cert crate doesn't support decoding CRLs as PEMs so created DER format for the tests, this was done via
```conosle
openssl crl -in verifier/data/tests/root_crl.pem -out verifier/data/tests/root_crl.der -outform DER
```

Reply:
Sorry my comment here was confusing.
I had originally downloaded the DER, converted it to PEM via OpenSSL and then converted it back to DER...

I've updated the readme such that it reads as download the DER, convert to PEM and I've double checked that doing those steps matches the repo contents.